### PR TITLE
Release Beta

### PR DESCRIPTION
**Pull request by kentcdodds**
**What**: Release the beta branch

**Why**: To drop node 8 and update to the latest of `@testing-library/dom`

**How**: Simple changes

**Checklist**:

- [ ] Documentation N/A
- [ ] Tests N/A
- [x] Ready to be merged

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,7 @@ dist
 # when working with contributors
 package-lock.json
 yarn.lock
+
+# cypress recordings during a test run are temporary files
+/cypress/videos/
+/cypress/screenshots/
```
Comment (NicholasBoll) [inline .gitignore]: 👍

File: package.json [hunk 2]
```diff
@@ -38,11 +38,11 @@
     "end-to-end",
     "e2e"
   ],
-  "author": "Kent C. Dodds <kent@doddsfamily.us> (http://kentcdodds.com/)",
+  "author": "Kent C. Dodds <me@kentcdodds.com> (https://kentcdodds.com)",
   "license": "MIT",
   "dependencies": {
     "@babel/runtime": "^7.8.4",
-    "@testing-library/dom": "^6.15.0",
+    "@testing-library/dom": "beta",
```
Comment (NicholasBoll) [inline package.json]: Curious what this will do? This is being merged into `master`, but uses the `beta` npm tag for `@testing-library/dom`?

**Comment by kentcdodds** [comment]
I'm preparing to release the beta of `@testing-library/dom` tomorrow. After that I'll merge this one as well. Anyone else have breaking changes we should make before doing that? @testing-library/cypress

**Comment by NicholasBoll** [comment]
Perhaps removing `query*` queries entirely?

**Comment by kentcdodds** [comment]
I'm actually ok with that. Should we just ditch the idea of deprecation messages?

**Comment by NicholasBoll** [comment]
I fixed the deprecation PR. It was a change to chai around `deep` vs `nested`

**Comment by NicholasBoll** [comment]
Also fallback of previous subject introduced by https://github.com/testing-library/cypress-testing-library/pull/108/commits/b58f74779499d1e09b87e566a95e0805f8254cb5 as part of #108

**Pull request merged**